.github/workflows/sync-main.yml

```
@@ -0,0 +1,48 @@
+name: automatic sync main branch from llvm-project to llvm
+
+on:
+  schedule:
+  - cron: '0 * * * *'
```

```suggestion
  - cron: '/10 * * * *'
```
Let's sync every 10 minutes rather than once an hour.